src/engine/dataset/index.ts

```diff
@@ -21,6 +20,7 @@ import { MutableOperation } from '../ops';
 import { MeldError } from '../MeldError';
 import { BaseStream, CountableRdf, QueryableRdfSource } from '../../rdfjs-support';
 import { AllMetadataSupport, Query } from '@rdfjs/types';
+import { uuid } from '../../util';
```

Comment:
Couldn't help noticing during this import of `uuid` from `util`.  Given that this commit was intended to replace uuid with SHA1, I'm wondering whether this is actually correct?

Reply:
This is correct. The switch from UUID is in the logical clock. Here it's being used for a guid.